programs/onreapp/src/instructions/buy_offer/state.rs

```diff
@@ -44,7 +45,7 @@ impl Default for BuyOffer {
             offer_id: 0,
             token_in_mint: Pubkey::default(),
             token_out_mint: Pubkey::default(),
-            time_segments: [BuyOfferTimeSegment::default(); MAX_SEGMENTS],
+            vectors: [BuyOfferVector::default(); MAX_VECTORS],
         }
     }
 }
```

Comment:
What do you guys @norbert-bodnar @rolnimar think, does it make sense to use `#[derive(Default)]` instead of explicitly providing the implementation? If I am not mistaken, semantics stay the same in this case.